lifecycle-operator/controllers/lifecycle/keptnapp/controller_test.go

```diff
@@ -236,6 +301,40 @@ func TestKeptnAppReconciler_deprecateAppVersions(t *testing.T) {
 	require.Equal(t, apicommon.StateDeprecated, keptnappversion.Status.Status)
 }
 
+func TestKeptnAppReconciler_deprecateAppVersionsError(t *testing.T) {
```

Comment:
Let's also test the happy path of the deprecation of the AppVersion, not only the error path